mordant/build.gradle.kts

```diff
@@ -34,6 +34,12 @@ kotlin {
             }
         }
 
+        val jvmMain by getting {
+            dependencies {
+                compileOnly(libs.graalvm.svm)
```

Comment:
Since we are only using the annotations, we don't need to depend on the library at runtime, meaning users of mordant won't have a new dependency on the classpath :)